src/main/java/com/hillel/items_exchange/service/UserService.java

```diff
@@ -57,4 +64,30 @@ public boolean existsByEmail(String email) {
     private UserDto mapUserToDto(User user) {
         return modelMapper.map(user, UserDto.class);
     }
+
+    public List<ChildDto> getChildren(User parent) {
+        return convertToDto(parent.getChildren(), ChildDto.class);
+    }
+
+    public void addChildren(User parent, List<ChildDto> childrenDtoToAdd) {
+        final List<Child> childrenToSave = new ArrayList<>(convertToModel(
+                childrenDtoToAdd, Child.class, ArrayList::new));
+        childrenToSave.forEach(child -> child.setUser(parent));
+        parent.getChildren().addAll(childrenToSave);
+        userRepository.save(parent);
```

Comment:
Why do you use `save` method here, but `saveAndFlush`after?

Reply:
Because when I use just "save" method tests are falls. I beleave it's because of changes maded by this methods flushes not fast enough and @ExpectedDataSet comparing fails. But in case "saveAndFlush" all work just fine.